### DISCUSSION TITLE
what muxer does srt support

### DISCUSSION THREAD

**Discussion by lotuschen** [Haivision/srt#2752] [Q&A] [answered]
Does srt need to use mpegts always?
Can I use other muxer at srt?

**Comment by lotuschen** [comment]
As rtp, mp4,...

**Reply by maxsharabayko** [reply, marked as answer]
SRT is content agnostic, it does not require using MPEG2-TS.

**Reply by lotuschen** [reply]
got it, thank you. @maxsharabayko
